scrapy/settings/default_settings.py

```diff
@@ -258,6 +258,7 @@
 SCHEDULER_DISK_QUEUE = 'scrapy.squeues.PickleLifoDiskQueue'
 SCHEDULER_MEMORY_QUEUE = 'scrapy.squeues.LifoMemoryQueue'
 SCHEDULER_PRIORITY_QUEUE = 'scrapy.pqueues.ScrapyPriorityQueue'
+SCHEDULER_DISK_QUEUE_CHUNKSIZE = 100000
```

Comment:
Maybe it was discussed already, but why is it an undocumented Scrapy setting, not just a constant? Is there a reason for user to change it? If yes, we should probably document it. If no, I'd just hardcode it - make it a constant, not a setting.

There can be disk queues which don't need chunksize (e.g. SQLite ones in queuelib), and I'm worried about having undocumented settings - this happened for some options in past, and it was quite hard to figure out what are they used for, if they are used in a real world, and if we can deprecate & remove them or not.

Reply:
> Is there a reason for user to change it?
 
There is no reason for user to change it and there was no way to do it before this PR. 
It was presented as, there aree several tests in `tests/test_squeues.py` testing behavior based on different values of this  argument.

I see several ways here:
1. remove option/argument together with tests 
2. provide documentation for option

Taking into account how much attention comparing to it's importance this option collected, I would prefer first variant.

Reply:
Agreed, +1 to the first approach.